libqpdf/QPDF_Name.cc

```diff
@@ -57,14 +55,12 @@ QPDF_Name::unparse()
 std::pair<bool, bool>
 QPDF_Name::analyzeJSONEncoding(const std::string& name)
 {
-    std::basic_string_view<unsigned char> view{
-        reinterpret_cast<const unsigned char*>(name.data()), name.size()};
-
     int tail = 0;       // Number of continuation characters expected.
     bool tail2 = false; // Potential overlong 3 octet utf-8.
     bool tail3 = false; // potential overlong 4 octet
     bool needs_escaping = false;
-    for (auto const& c: view) {
```

Comment:
We can simply change c -> ch and  view -> name here and stick with a range based for loop.